### ISSUE TITLE
Creating Pull Request in VS2019 fails

### ISSUE THREAD

**Issue by webteckie** [MistyKuu/bitbucket-for-visual-studio#121]
Creating a Pull Request in latest Visual Studio 2019 Enterprise (extension version 1.41.1) fails with error message:

```Errors encountered while adding some reviewers to this pull request```

It did allow me to correctly select and add the reviewers.

From log:

```
2019-06-18 13:39:09,662 [1] ERROR BitBucket.REST.API.Wrappers.BitbucketRestClientBase [(null)] - Error in request: {"errors":[{"context":"reviewers","message":"Errors encountered while adding some reviewers to this pull request.","exceptionName":"com.atlassian.bitbucket.pull.InvalidPullRequestReviewersException","reviewerErrors":[{"context":"Saivenkat XXX","message":"Saivenkat XXX is not a user.","exceptionName":null},{"context":"Benjamin XXX","message":"Benjamin XXX is not a user.","exceptionName":null},{"context":"Shilpa XXX","message":"Shilpa XXX is not a user.","exceptionName":null}],"validReviewers":[]}]}
2019-06-18 13:39:09,675 [1] ERROR GitClientVS.Infrastructure.Extensions.ViewModelsExtensions [(null)] - BitBucket.REST.API.Exceptions.RequestFailedException: Errors encountered while adding some reviewers to this pull request.
   at BitBucket.REST.API.Wrappers.BitbucketRestClientBase.CheckResponseStatusCode(IRestResponse response)
   at BitBucket.REST.API.Wrappers.BitbucketRestClientBase.<ExecuteTaskAsync>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at BitBucket.REST.API.Clients.Enterprise.EnterprisePullRequestsClient.<CreatePullRequest>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitClientVS.Services.BitbucketService.<CreatePullRequest>d__39.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitClientVS.Infrastructure.ViewModels.CreatePullRequestsViewModel.<CreateNewPullRequest>d__85.MoveNext()
2019-06-18 13:40:22,122 [1] INFO  BitBucket.REST.API.Wrappers.BitbucketRestClientBase [(null)] - Calling ExecuteTaskAsync. BaseUrl: https://stash.ep.se/rest/api/1.0 Resource: projects/nas/repos/episerver.reporting/pull-requests
2019-06-18 13:40:22,849 [1] ERROR BitBucket.REST.API.Wrappers.BitbucketRestClientBase [(null)] - Error in request: {"errors":[{"context":"reviewers","message":"Errors encountered while adding some reviewers to this pull request.","exceptionName":"com.atlassian.bitbucket.pull.InvalidPullRequestReviewersException","reviewerErrors":[{"context":"Saivenkat XXX","message":"Saivenkat XXX is not a user.","exceptionName":null},{"context":"Benjamin XXX","message":"Benjamin XXX is not a user.","exceptionName":null},{"context":"Shilpa XXX","message":"Shilpa XXX is not a user.","exceptionName":null}],"validReviewers":[]}]}
2019-06-18 13:40:22,853 [1] ERROR GitClientVS.Infrastructure.Extensions.ViewModelsExtensions [(null)] - BitBucket.REST.API.Exceptions.RequestFailedException: Errors encountered while adding some reviewers to this pull request.
   at BitBucket.REST.API.Wrappers.BitbucketRestClientBase.CheckResponseStatusCode(IRestResponse response)
   at BitBucket.REST.API.Wrappers.BitbucketRestClientBase.<ExecuteTaskAsync>d__9.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at BitBucket.REST.API.Clients.Enterprise.EnterprisePullRequestsClient.<CreatePullRequest>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitClientVS.Services.BitbucketService.<CreatePullRequest>d__39.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitClientVS.Infrastructure.ViewModels.CreatePullRequestsViewModel.<CreateNewPullRequest>d__85.MoveNext()
```

**Comment by MistyKuu** [comment]
this is bitbucket server right?

**Comment by MistyKuu** [comment]
Should be fixed in 1.41.2
